### ISSUE TITLE
[Wiki] Visualeditor

### ISSUE THREAD

**Issue by ghost** [Electrical-Age/ElectricalAge#570]
Can we install the visualeditor on the wiki?  (I can do it if you like)

**Comment by metc** [comment]
Hey, Yes, I can install that plugin if you need it, but it will no be
available before this weekend... Sorry for that I have a limited access to
the Internet right now.

**Comment by ghost** [comment]
Cool, thanks. Take your time. I don't need it tomorrow.  

Visualeditor is one of the trickier extensions to do. So if it's no problem for you, I bet you can do anything ;-)

**Comment by metc** [comment]
I am not sure we can make running this extension on a shared hosting server. It does require a `Parsoid` server and I cannot install any custom packages on that server...
@KimBruning Have you already installed this extension or have you any experience with it ?

**Comment by ghost** [comment]
Yes and Yes. Installed, have experience. If you're on shared hosting now, might I suggest moving the wiki to madoka.brage.info? (see also the other thread :-) )
